app/views/shell/_top.html.erb

```diff
@@ -50,6 +50,7 @@
       <meta property="forem:name" content="<%= community_name %>" />
       <meta property="forem:logo" content="<%= optimized_image_url(SiteConfig.logo_png, width: 512, fetch_format: "png") %>" />
       <meta property="forem:domain" content="<%= SiteConfig.app_domain %>" />
+      <meta property="forem:releasedate" content="<%= ENV["RELEASE_FOOTPRINT"] %>" />
```

Comment:
We already have [this data below in our body tag](https://github.com/forem/forem/pull/10834/files#diff-54c7373a1a9cafdb686788688f850f3a4469b23c6b6d48e39751a02b54795caaR64) Duplicating it feels unnecessary, should we remove the one below in favor of this one?